src/colored_trees.jl

```diff
@@ -18,26 +18,32 @@ See also [`BicoloredRootedTree`](@ref), [`rootedtree`](@ref).
   SIAM Journal on Numerical Analysis 34.5 (1997): 1926–1947.
   [DOI: 10.1137/S0036142995292128](https://doi.org/10.1137/S0036142995292128)
 """
-struct ColoredRootedTree{T<:Integer, V<:AbstractVector{T}, C<:AbstractVector} <: AbstractRootedTree
-  level_sequence::V
-  color_sequence::C
-  iscanonical::Bool
-
-  function ColoredRootedTree(level_sequence::V, color_sequence::C, iscanonical::Bool=false) where {T<:Integer, V<:AbstractVector{T}, C<:AbstractVector}
-    new{T, V, C}(level_sequence, color_sequence, iscanonical)
-  end
+struct ColoredRootedTree{T <: Integer, V <: AbstractVector{T}, C <: AbstractVector} <:
+       AbstractRootedTree
+    level_sequence::V
+    color_sequence::C
+    iscanonical::Bool
+
+    function ColoredRootedTree(level_sequence::V, color_sequence::C,
+                               iscanonical::Bool = false) where {T <: Integer,
+                                                                 V <: AbstractVector{T},
+                                                                 C <: AbstractVector}
+        new{T, V, C}(level_sequence, color_sequence, iscanonical)
+    end
 end
 
-
 """
     BicoloredRootedTree{T<:Integer}
 
 Representation of bicolored rooted trees.
 
 See also [`ColoredRootedTree`](@ref), [`RootedTree`](@ref), [`rootedtree`](@ref).
 """
-const BicoloredRootedTree{T<:Integer, V<:AbstractVector{T}, C<:AbstractVector{Bool}} = ColoredRootedTree{T, V, C}
-
+const BicoloredRootedTree{T <: Integer, V <: AbstractVector{T}, C <: AbstractVector{Bool}} = ColoredRootedTree{
+                                                                                                               T,
+                                                                                                               V,
+                                                                                                               C
+                                                                                                               }
```

Comment:
The part on the right looks really weird... :shrug:

Reply:
https://github.com/domluna/JuliaFormatter.jl/issues/607

yeah...

we gotta keep pushing it 😅  Definitely we've brought the formatter to places it hasn't gone to before: https://github.com/domluna/JuliaFormatter.jl/issues/610